earthfile2llb/listener.go

```diff
@@ -158,6 +160,15 @@ func (l *listener) ExitFromStmt(c *parser.FromStmtContext) {
 	for i, ba := range buildArgs.Args {
 		buildArgs.Args[i] = l.expandArgs(ba, true)
 	}
+
+	if imageName == "LOCAL" {
+		imageName = "alpine:3.13"
```

Comment:
I would move this hack to the converter - it helps keep the listener focused on syntax only. Perhaps there needs to be a separate call for `FromLocal` in the converter.

Reply:
Good idea on the `FromLocal` function in the converter.